sherpa/utils/__init__.py

```diff
@@ -1247,6 +1247,10 @@ def get_keyword_names(func, skip=0):
 def get_keyword_defaults(func, skip=0):
     """Return the keyword arguments and their default values.
 
+    Note that a similar function `sherpa.plot.backend_utils.get_keyword_defaults`
+    exits, which differs form this one in that is deal with keyword-only
```

Comment:
Is this meant to be something like>

```suggestion
    exits, which differs from this one in that it deals with keyword-only
```